src/openrct2/actions/PlacePeepSpawnAction.hpp

```diff
@@ -55,6 +55,7 @@ DEFINE_GAME_ACTION(PlacePeepSpawnAction, GAME_COMMAND_PLACE_PEEP_SPAWN, GameActi
 
         auto res = std::make_unique<GameActionResult>();
         res->Expenditure = ExpenditureType::LandPurchase;
+        // TODO: Suspicious z/8 to transform Coords into Coords
         res->Position = CoordsXYZ{ _location.x, _location.y, _location.z / 8 };
```

Comment:
Just make the change (remove the /8) as it will have been a mistake when setting this up. There isn't any issues with change the res->Position it only changes where error messages are.

Reply:
Alright